java/animate-3d-graphic/README.md

```diff
@@ -1,58 +1,52 @@
-# Animate 3D Graphic
+# Animate 3D graphic
 
-Animate a graphic's position and orientation and follow it with the camera.
+An `OrbitGeoElementCameraController` follows a graphic while the graphic's position and rotation are animated.
 
-![Animate 3D Graphic App](animate-3d-graphic.png)
+![Image of animate 3D graphic](animate-3d-graphic.png)
 
 ## How to use the sample
 
 Animation Controls (Top Left Corner):
+
 * Select a mission -- selects a location with a route for plane to fly.
 * Mission progress -- shows how far along the route the plane is. Slide to change keyframe in animation.
 * Play -- toggles playing and stopping the animation.
 * Follow -- toggles camera following plane.
 
 Speed Slider (Top Right Corner):
+
```

Comment:
let's keep this all on one line rather than split it

Reply:
How do you mean? As in
`* Speed Slider (Top Right Corner): Controls speed of animation.` ?

Reply:
yeh!

Reply:
Does this not conflict with the same type of descriptor `Animation Controls (Top Left Corner)`? Happy to change it,  but that looks more confusing to me personally

Reply:
oh, I see what you mean now, missed that before. Sure, that's fine to leave as is then. Thanks!